dipy/denoise/nlmeans_block.pyx

```diff
@@ -0,0 +1,358 @@
+cimport cython
```

Comment:
This file seems not optimized properly. 
Use 
```cython -a nlmeans_block.pyx
google-chrome nlmeans_block.html
```
to see which areas are not optimized.
Look for functions inside loops that are still yellow.